TableCell/index.jsx

```diff
@@ -16,11 +16,7 @@ const TableCellStateless = ({
     hovered,
   });
   return (
-    <div
-      onMouseEnter={onMouseEnter}
-      onMouseLeave={onMouseLeave}
-      style={style}
-    >
+    <div onMouseEnter={onMouseEnter} onMouseLeave={onMouseLeave} style={style}>
```

Comment:
Good call. I'll note this one down and pending any changes we might make to `CloseIcon`, I'll go ahead and revert this one. 👍